app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt

```diff
@@ -438,6 +439,7 @@ class PreviewSliderFragment : Fragment(), FileInfoActionsView.OnItemClickListene
                 removeFileInSlider()
                 showSnackbar(getString(R.string.snackbarMoveTrashConfirmation, currentFile.name))
                 mainViewModel.deleteFileFromHome.value = true
+                setBackNavigationResult(DELETE_FILE_KEY, currentFile.id, withPopBackStack = false)
```

Comment:
Maybe use a SingleLiveEvent instead of tweaking a existing system that is not designed for this use case ?

Reply:
Done